src/libsync/owncloudpropagator.cpp

```diff
@@ -323,6 +323,27 @@ bool PropagateItemJob::checkForProblemsWithShared(int httpStatusCode, const QStr
     return false;
 }
 
+void PropagateItemJob::readFileIdHeaders(const QNetworkReply &reply)
```

Comment:
Any special reason to use reference for a qobject which was on heap before? (i don't think we do that anywhere else in the client)